jenkins.py

```diff
@@ -324,9 +324,10 @@ def read_and_post_job_metrics(module_config, url, job_name, last_timestamp):
     resp_obj = get_response(job_url, 'jenkins', module_config)
     extra_dimensions = {}
     extra_dimensions['Job'] = job_name
-    if resp_obj and resp_obj['builds']:
-        for i in xrange(len(resp_obj['builds'])):
-            build_url = job_url + str(resp_obj['builds'][i]['number']) + '/'
+    builds = resp_obj.get('builds', None)
```

Comment:
Do you know for sure that resp_obj is not None or a dictionary at this point?  If it is None or some other type this will raise an Exception.